qiskit/pulse/commands/parametric_pulses.py

```diff
@@ -0,0 +1,292 @@
+             = Gaussian + 1j * beta * (-x / sigma) [Gaussian]
+
+    where 'Gaussian' is:
+        Gaussian(x, amp, sigma) = amp * exp( -(1/2) * (x - duration/2)^2 / sigma^2) )
+
+    Ref:
+        [1] Gambetta, J. M., Motzoi, F., Merkel, S. T. & Wilhelm, F. K.
+        Analytic control methods for high-fidelity unitary operations
+        in a weakly nonlinear oscillator. Phys. Rev. A 83, 012308 (2011).
+        [2] F. Motzoi, J. M. Gambetta, P. Rebentrost, and F. K. Wilhelm
+        Phys. Rev. Lett. 103, 110501 – Published 8 September 2009
+    """
+
+    def __init__(self,
+                 duration: int,
+                 amp: complex,
+                 sigma: float,
+                 beta: int):
+        """Initialize the drag command.
+
+        Args:
+            duration: Pulse length in terms of the the sampling period `dt`.
+            amp: The amplitude of the Drag envelope.
+            sigma: A measure of how wide or narrow the Gaussian peak is; described mathematically
+                   in the class docstring.
+            beta: The correction amplitude.
+        """
+        validate_params(amp=amp, sigma=sigma, beta=beta)
+        self._amp = complex(amp)
+        self._sigma = sigma
+        self._beta = beta
+        super().__init__(duration=duration)
+
+    @property
+    def amp(self):
+        return self._amp
+
+    @property
+    def sigma(self):
+        return self._sigma
+
+    @property
+    def beta(self):
+        return self._beta
+
+    def get_sample_pulse(self) -> SamplePulse:
+        return drag(duration=self.duration, amp=self.amp, sigma=self.sigma,
+                    beta=self.beta, zero_ends=False)
+
+
+class ConstantPulse(ParametricPulse):
+    """
+    A simple constant pulse, with an amplitude value and a duration:
+
+        f(x) = amp    ,  0 <= x < duration
+        f(x) = 0      ,  elsewhere
+    """
+
+    def __init__(self,
+                 duration: int,
+                 amp: complex):
+        """
+        Initialize the constant valued pulse command.
+
+        Args:
+            duration: Pulse length in terms of the the sampling period `dt`.
+            amp: The amplitude of the constant square pulse.
+        """
+        validate_params(amp=amp)
+        self._amp = complex(amp)
+        super().__init__(duration=duration)
+
+    @property
+    def amp(self):
+        return self._amp
+
+    def get_sample_pulse(self) -> SamplePulse:
+        return constant(duration=self.duration, amp=self.amp)
+
+
+def validate_params(**kwargs):
+    """
+    Raise a PulseError if the parameters passed are not valid.
+    """
+    if 'amp' in kwargs:
+        if abs(kwargs['amp']) > 1.:
+            raise PulseError("The amplitude norm must be less than 1, "
+                             "found: {}".format(abs(kwargs['amp'])))
+    if 'sigma' in kwargs:
+        if kwargs['sigma'] <= 0:
+            raise PulseError("Sigma must be greater than 0.")
+    if 'beta' in kwargs:
```

Comment:
Restrict beta more

Reply:
Change `PulseError` comment to less than or equal to 1, I believe?